AppCenter/AppCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -9,13 +9,17 @@ @implementation MSDBStorage
 - (instancetype)initWithSchema:(MSDBSchema *)schema version:(NSUInteger)version filename:(NSString *)filename {
   if ((self = [super init])) {
     _dbFileURL = [MSUtility createFileAtPathComponent:filename withData:nil atomically:NO forceOverwrite:NO];
-    _maxPageCount = [MSDBStorage numberOfPagesInBytes:kMSDefaultDatabaseSizeInBytes];
+    _maxSizeInBytes = kMSDefaultDatabaseSizeInBytes;
 
     // If it is custom SQLite library we need to turn on URI filename capability.
     sqlite3_config(SQLITE_CONFIG_URI, 1);
 
     // Execute all initialize operation with one database instance.
-    [self executeQueryUsingBlock:^int(void *db) {
+    int result;
+    sqlite3 *db = [MSDBStorage openDatabaseAtFileURL:self.dbFileURL withResult:&result];
```

Comment:
Seems `result` is ignored. Can you pass `nil` instead of a reference?

Reply:
It's logged inside (just like before). No without refactoring the method to accept `nil`. Is it really necessary?

Reply:
No. 😄